translations/ja/LC_MESSAGES/messages.po

```diff
@@ -161,7 +161,7 @@ msgstr ""
 
 #: /content/includes/menu.yaml
 msgid "About"
-msgstr ""
+msgstr "AMP とは"
```

Comment:
Korean translation are also use different translation for batter word.
For example, if translate `about` to directly in Korean that is `대하여`, but it's too difficult to know about what for that menu.
So i'll add nouns so that is `AMP에 대하여`, Japanese also same for this problem.

But I think you can change this one to `AMPについて`.